src/components/Header/DesktopDropdown.js

```diff
@@ -43,7 +43,7 @@ const DesktopMenuDropdownContainer = styled(MenuItem)`
 
       &:hover {
         background: ${props => props.theme.colors.vibrant};
-        color: ${props => props.theme.colors.textLight};
+        color: #007F56;
```

Comment:
putting new color from docs

Reply:
Cool! I've hit 'resolve' on the convo above :+1:

So we've got 2 known issues, like we were talking about on Slack:
1. We now have a hover state on the DropdownName for the dark theme and don't have one for the light theme
2. We also know that this colour contrast is problematic.

But we don't need to resolve that in this PR. I reckon we just need to make a note for later. So I'm gonna create a new trello ticket for this stuff, to go through Antonas, if that's cool with you?

Reply:
very cool!